xcube-dev/xcube-viewer
# Ease development of server-side viewer panels

Changes:
- Added developer reference docs generated from derived state's property descriptions
- New module `ext` that deals solely with the server-side contributions
- Introduced new derived state for server-side contributions
- Updated the demo panel "2D Histogram", it is still lacking some useful features like showing progress, disablement, and some info text

To verify
- make sure dev reference works: extensions and derived state is documented
- make sure the demo panels work, please leave comments with possible enhancements or ideas for other panels

See https://github.com/xcube-dev/xcube/pull/1092
